models/team.model.js

```diff
@@ -6,8 +6,7 @@ const Constants = require("../constants/general.constant");
 const TeamSchema = new mongoose.Schema({
     name: {
         type: String,
-        unique: true,
```

Comment:
Do we not want to make this unique (so that we create a fast index on this)?

Reply:
I just feel that _id being index might be more useful.